sdk/ai/Azure.AI.AgentServer/src/Contracts/Azure.AI.AgentServer.Contracts.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);OPENAI001;CS1573;CS1574;CS1591;AZC0007;AZC0012;AZC0014;AZC0015;AZC0030;AZC0031;AZC0033;AZC0034;SA1649;SA1402;SA1636;</NoWarn>
```

Comment:
There are a lot of suppressions here that I'm uncomfortable with.   I'd like @KrzysztofCwalina's perspective here on what exceptions we intend to make on both a short and long term basis to help inform the right path forward.